src/common/transformations/tests/common_optimizations/mark_precision_sensitive_shapeof_subgraphs_test.cpp

```diff
@@ -0,0 +1,371 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/mark_precision_sensitive_shapeof_subgraphs.hpp"
+
+#include <gtest/gtest.h>
+
+#include <memory>
+#include <openvino/opsets/opset5.hpp>
+#include <openvino/opsets/opset8.hpp>
+#include <openvino/pass/manager.hpp>
+#include <string>
+#include <transformations/init_node_info.hpp>
+#include <transformations/utils/utils.hpp>
+#include <vector>
+
+#include "common_test_utils/graph_comparator.hpp"
+#include "transformations/rt_info/disable_fp16_compression.hpp"
+
+using namespace testing;
+using namespace ov;
+
+TEST(TransformationTests, MarkEntireShapeSubgraphs_trivial_case) {
+    // check that marking does not leak in trivial case
+    std::shared_ptr<ov::Model> model(nullptr), model_ref(nullptr);
```

Comment:
`ov::Model` -> `Model` because `using namespace ov;` and other places

Reply:
done